Login App/RandomViewController.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  RandomViewController.swift
+//  Login App
+//
+//  Created by user on 13/09/17.
+//  Copyright © 2017 User. All rights reserved.
+//
+
+import UIKit
+
+class RandomViewController: UIViewController {
+    
+    
+    @IBOutlet weak var name: UILabel!
+    
+    @IBOutlet weak var emailId: UILabel!
+    
+    @IBOutlet weak var dateOfBirth: UILabel!
+    
+    @IBOutlet weak var imageView: UIImageView!
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        
+        self.navigationController?.navigationBar.topItem?.title = "Random";
+        
+    }
+    
+    @IBAction func addRandomUser(_ sender: UIButton) {
+        
+        let Request: String = "https://randomuser.me/api/"
+        
+        guard let url = URL(string: Request) else {
+            
+            return
+        }
+        
+        let urlRequest = URLRequest(url: url)
+        
+        let session = URLSession.shared
+        
+        let task = session.dataTask(with: urlRequest, completionHandler: {(data:Data?, response: URLResponse? , error: Error?) -> Void in
+            
+            guard error == nil else {
+                
+                print("Error \(String(describing: error))")
+                
+                return
+                
+            }
+            
+            guard let responseData = data else {
+                
+                print("Response data is empty)")
+                
+                return
+                
+            }
+            
+            print("Response data \(responseData)")
+            
+            do {
+                
+                guard let jsonDictionary = try JSONSerialization.jsonObject(with: responseData, options: [] ) as? [String: AnyObject] else {
+                    
+                    return
+                    
+                }
+                
+                print("The data the user having is: \(jsonDictionary)")
+                
+                
+                if let result = jsonDictionary["results"] as? [[String:Any]] {
+                    
+                    let  array = result[0]
```

Comment:
What is this array? Have proper naming conventions.

Reply:
I changed the name of array to userDetails and pushed the code